extensions/amp-story/1.0/amp-story.js

```diff
@@ -347,13 +347,14 @@ export class AmpStory extends AMP.BaseElement {
     // prerendering, because of a height incorrectly set to 0.
     this.mutateElement(() => {});
 
-    if (
+    const needsDvhPolyfill =
```

Comment:
All this needs to go below the lines that contain `data-story-supports-landscape` but I wouldn't split them like this because it's harder to trace back